CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+First of all, Welcome, and Thank You!
+
+Graphite has been a community project for years, and it only gets better by the efforts of people who volunteer.  We sincerely appreciate that you care enough to see a problem, take the time to make this project better.  Whether it's in the form of a bug report, code fix, or documentation improvement, we - the Graphite team - appreciate your help.
+
+This is a collection of practices that will make it easier for us to accept your contributions.
+
+
+### Issues
+
+While not mandatory, opening an Issue against the appropriate repo is seldom a bad thing.  We prefer that you have searched through the existing issues.  It's entirely possible that the issue has already been reported.  It may have already been solved.
+
+If your issue in in-fact new, please write a descriptive report.  Terse but complete is best.  You should describe:
+
+* What you attempted
+* Expected Result
+* Witnessed Result
+* System Details
+  * OS / Version
+  * Python Version
+  * Django Version (a frequent cause of issues)
+  * Graphite Component Versions (or commit hash if running directly from the repo)
+
+
+### Tests
+
+We would absolutely love to improve our test coverage.  If you see a gap, and can write a test for that, we'd eat that up.
+
+
+### Code
+
+If you see a mistake, have a feature, or a performance gain, we'd love to see it.  It's _strongly_ encouraged for contributions that aren't already covered by tests to come with them.  We're not trying to foist work on to you, but it makes it much easier for us to accept your contributions.
```

Comment:
I just meant that we have a dedicated **Tests** section literally preceding this one.